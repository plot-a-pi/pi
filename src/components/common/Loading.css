.Loading{
  position: fixed;
  width: 100%;
  img{
    display: block;
    margin: auto;
    width: 300px;

  }
  top: 30%;
  margin: auto;

  @media screen and (min-width: 1200px) {
    width: 50%;
    top: 40%;
    left: 2%;
    img{
      width: 25%;
    }
  }

}