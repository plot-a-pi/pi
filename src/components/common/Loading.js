import React from 'react';
import styles from './Loading.css';
import loading from '../../assets/loading.gif';

const Loading = () => {
  return (
    <div className={styles.Loading}>
      <img src={loading} />
    </div>
  );
};

export default Loading;
