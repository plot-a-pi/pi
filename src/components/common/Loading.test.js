import React from 'react';
import { shallow } from 'enzyme';
import Loading from './Loading';

describe('Loading Component', () => {
  it('should render a Loading', () => {
    const wrapper = shallow(<Loading/ >);
    expect(wrapper).toMatchSnapshot();
  });
});
