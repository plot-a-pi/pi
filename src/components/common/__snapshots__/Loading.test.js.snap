// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component should render a Loading 1`] = `
<div
  className="Loading"
>
  <img
    src="/path/image"
  />
</div>
`;
